toolchain/check/diagnostic_emitter.cpp

```diff
@@ -161,4 +174,11 @@ auto DiagnosticEmitter::ConvertArg(llvm::Any arg) const -> llvm::Any {
   return DiagnosticEmitterBase::ConvertArg(arg);
 }
 
+// static
```

Comment:
We typically don't add these kinds of comments, I'd suggest not doing it here for consistency.
```suggestion
```

Reply:
Done.

Reply:
Done.